Sources/Endpoints/TestFlight/Apps/GetAppInformation.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  ReadAppInformation.swift
+//  AppStoreConnect-Swift-SDK
+//
+//  Created by Pascal Edmond on 20/11/2018.
+//
+
+import Alamofire
+
+extension APIEndpoint {
+    
+    /// Get information about a specific app.
+    ///
+    /// - Parameters:
+    ///   - id: An opaque resource ID that uniquely identifies the resource.
+    ///   - fields: Fields to return for included related types.
+    ///   - relationships: Relationship data to include in the response.
+    ///   - limits: Number of included related resources to return.
+    public static func app(
+        with id: String,
+        select fields: [ReadAppInformation.Field]? = nil,
+        include relationships: [ReadAppInformation.Relationship]? = nil,
+        limits: [ReadAppInformation.Limit]? = nil) -> APIEndpoint<AppResponse> {
+        var parameters = [String: Any]()
+        if let fields = fields { parameters.merge(with: fields) }
+        if let relationships = relationships { parameters.merge(with: relationships) }
+        if let limits = limits { parameters.merge(with: limits) }
+        return APIEndpoint<AppResponse>(path: "apps/\(id)", method: .get, parameters: parameters)
```

Comment:
Can we write tests for this to verify the outcoming request?

Reply:
Let me know if it's missing something : [ReadAppInformationTests.swift](https://github.com/AvdLee/appstoreconnect-swift-sdk/blob/dabddc3c7833ab8b4c7f64ce54982747438c926b/Example/CocoaPods-AppStoreConnect-Swift-SDK/Tests/ReadAppInformationTests.swift)